### PR TITLE
Feature/add get templates endpoint

### PR DESCRIPTION
**Pull request by darkmavis1980**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -24,12 +24,12 @@
     "no-unused-vars": "warn"
   },
   "globals": {
-    "$": true,
-    "document": true,
-    "window": true,
-    "navigator": true,
-    "localStorage": true,
-    "jQuery": true,
+    "$": false,
```
Comment (smcelhinney) [inline .eslintrc]: Curious as to why these are changed?

Comment (smcelhinney) [reply .eslintrc]: If they're not required (browser-only) then you can remove them altogether.

Comment (darkmavis1980) [reply .eslintrc]: Because it's a node script, so you don't need any of the browsers feature, or jQuery

Comment (darkmavis1980) [reply .eslintrc]: Ah perfect, I'll remove them then!

**Pull request merged**